### PR TITLE
sdp: refactor jingle2media

### PR DESCRIPTION
**Pull request by fippo**
using more specific selectors

### CODE REVIEW

File: modules/xmpp/SDP.js [hunk 1]
```diff
@@ -699,30 +719,9 @@ SDP.prototype.jingle2media = function(content) {
                 hdrExt.getAttribute('uri')}\r\n`;
     });
 
-    content
-        .find(
-            '>transport[xmlns="urn:xmpp:jingle:transports:ice-udp:1"]'
-                + '>candidate')
-        .each((_, transport) => {
```
Comment (fippo) [inline modules/xmpp/SDP.js:706]: oh, transport here was the wrong name. Well, drive-by fix, thanks linter ;-)

**Comment by fippo** [comment]
@hristoterezov @paweldomas wanna take a look? You seem to like sdp things ;-)

**Comment by saghul** [comment]
I hear @jallamsetty1 likes SDP :-)
